features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/Configuration/Configuration.jsx

```diff
@@ -190,6 +199,23 @@ class Configuration extends React.Component {
             return apiTransport.includes(type);
         }
     }
+    setSecurityScheme = (updatedSecurityScheme) => {
+        this.setState((oldState) => {
+            let { securityScheme } = oldState;
+            securityScheme = updatedSecurityScheme;
+            return { securityScheme };
```

Comment:
I think we can simply set this like 

this.setState({securityScheme: updatedSecurityScheme})

Reply:
done